extensions/resource-deployment/src/ui/deployClusterWizard/pages/summaryPage.ts

```diff
@@ -183,6 +184,11 @@ export class SummaryPage extends WizardPageBase<DeployClusterWizard> {
 						label: localize('deployCluster.DomainServiceAccountUserName', "Service account username"),
 						defaultValue: this.wizard.model.getStringValue(VariableNames.DomainServiceAccountUserName_VariableName),
 						labelCSSStyles: { fontWeight: FontWeight.Bold }
+					}, {
+						type: FieldType.ReadonlyText,
+						label: localize('deployCluster.Realm', "Realm"),
```

Comment:
There is a localizedConstants file so it'd be nice to just put this there (especially since you use it twice here)